app/src/androidTest/java/com/github/sdp_begreen/begreen/firebase/FirebaseDBTest.kt

```diff
@@ -31,6 +32,8 @@ import kotlin.test.junit.JUnitAsserter.fail
 @LargeTest
 class FirebaseDBTest {
```

Comment:
This test class is getting a bit too big. It might be more useful to have a test file dedicated to bin placement, for example.

Reply:
You're right, we were planning to split `FirebaseDB` in several services (just like for the meetings) to gain in readability. Thanks for your help!